homeassistant/components/romy/strings.json

```diff
@@ -0,0 +1,52 @@
+{
+  "config": {
+    "flow_title": "{name}",
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      },
+      "unlock_http_interface": {
+        "title": "Local Http Interface is locked",
+        "data": {
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "password": "Please provide password (8 characters, see QR Code under the dustbin)."
```

Comment:
I am not super familiar with all the config flow fields and how they are displayed -- would it be nicer if this were `description` for the entire the step (now that its pulled out to be its own step) or does this look ok as a data description?

Reply:
this is for the password data field description changed it now from
![pw_old](https://github.com/home-assistant/core/assets/2132412/fa613392-5b82-4715-92af-20b9c387589d)
to:
![pw_new](https://github.com/home-assistant/core/assets/2132412/93a5866c-cc55-4e9e-9282-9b4f31d9bbed)